ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/dps-interfaces.schema.yml

```diff
@@ -0,0 +1,61 @@
+# Copyright (c) 2023 Arista Networks, Inc.
+# Use of this source code is governed by the Apache License 2.0
+# that can be found in the LICENSE file.
+# yaml-language-server: $schema=../../../../plugins/plugin_utils/schema/avd_meta_schema.json
+# Line above is used by RedHat's YAML Schema vscode extension
+# Use Ctrl + Space to get suggestions for every field. Autocomplete will pop up after typing 2 letters.
+type: dict
+keys:
+  dps_interfaces:
+    type: list
+    primary_key: name
+    min_length: 1
+    max_length: 1
+    items:
+      type: dict
+      keys:
+        name:
+          type: str
+          description: Only Dps1 is configurabled.
+          valid_values:
+            - Dps1
+        description:
+          type: str
+        ip_address:
+          type: str
+          description: IPv4 address/mask
+        flow_tracker:
+          type: dict
+          keys:
+            sampled:
```

Comment:
show we also add sampled egress

```
eos-1(config-if-Dp1)#sh active 
interface Dps1
   description Test DPS Interface
   flow tracker hardware FT-HW
   flow tracker sampled test
   flow tracker sampled egress test_egress
```

Reply:
for now even `sampled` was not required, only `hardware` - we can probably add this later if needed.